docs/pguidance/agent-k8s-install-kubectl.md

```diff
@@ -0,0 +1,128 @@
+* SignalFx access token. See [Smart Agent Access Token](../../../_sidebars-and-includes/smart-agent-access-token.html).
+* Your SignalFx realm. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+
+## Configure kubectl for the Smart Agent
+
+To install the Smart Agent using kubectl, you need to create a
+Kubernetes secret for your access token and update settings in the Smart Agent's configuration files.
+
+1. To create the Kubernetes secret, log in to the host from which you run kubectl.
+2. Run the following command to create the Kubernetes secret called signalfx-agent, substituting `<access_token>` with
+   your SignalFx access token:
+
+   ```
+   kubectl create secret generic --from-literal access-token=<access_token> signalfx-agent
+   ```
+
+3. On the same host, download the following configuration files from the
+   [Kubernetes Deployments](https://github.com/signalfx/signalfx-agent/tree/master/deployments/k8s) area in GitHub:
+
+   | File                      | Description                                                                                    |
+   |:--------------------------|:-----------------------------------------------------------------------------------------------|
+   | `clusterrole.yaml`        | Configuration settings for the ClusterRole cluster-admin. This file doesn't require an update. |
+   | `clusterrolebinding.yaml` | Configuration settings for Kubernetes role-based access control (**RBAC**)                     |
+   | `configmap.yaml`          | Cluster configuration settings                                                                 |
+   | `daemonset.yaml`          | Daemonset configuration settings                                                               |
+   | `serviceaccount.yaml`     | Configuration settings for Kubernetes Service Accounts. This file doesn't require an update.   |
+
+4. Update `configmap.yaml`:
+   - Cluster name: For each of your Kubernetes clusters, replace `MY-CLUSTER` with a unique cluster name.
+   - Realm: Update the value of `signalFxRealm` with the name of your SignalFx realm.
+   - To avoid sending docker and cadvisor metrics from some containers,
+     update the `datapointsToExclude` property. To learn more, see [Filtering](https://docs.signalfx.com/en/latest/integrations/agent/filtering.html#filtering).
+5. In the clusterrolebinding.yaml file, update `MY_AGENT_NAMESPACE` or the service account token reference with the Smart
+   Agent namespace in which you're deploying the agent.
+
+6. Update the daemonset.yaml file:
+
+   - For RBAC-enabled clusters, add the permissions required for the Smart Agent.
+
+   - For Rancher nodes, ensure that the Docker engine proxy is configured so that it can pull the `signalfx-agent` Docker image from `quay.io`.
+     To learn more, see the Rancher v1.6 or Rancher v2.x documentation regarding proxy configuration.
+
+   - Update the **cAdvisor** monitor configuration to use port 9344:
```

Comment:
is this still relevant? cadvisor is no longer used on k8s to my knowledge

Reply:
I'll chec,.

Reply:
See [Smart Agent Install for Kubernetes using kubectl](https://docs.signalfx.com/en/latest/integrations/kubernetes/k8s-advanced-installation.html#smart-agent-install-for-kubernetes-using-kubectl). Step 2 point 3 refers to it. Is it required, optional, something else? Please advise.